src/bundle/DependencyInjection/Compiler/LexikAuthorizationHeaderBridgePass.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\Rest\DependencyInjection\Compiler;
+
+use Ibexa\Contracts\Rest\Security\AuthorizationHeaderRESTRequestMatcher;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+final class LexikAuthorizationHeaderBridgePass implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container): void
+    {
+        if (!$container->hasDefinition(AuthorizationHeaderRESTRequestMatcher::class)) {
+            return;
+        }
+
+        if (!$container->hasDefinition('lexik_jwt_authentication.extractor.authorization_header_extractor')) {
+            return;
+        }
```

Comment:
You can make this into 1 `||` expression to reduce number of returns.
Sth like:
```suggestion
        if (
            !$container->hasDefinition(AuthorizationHeaderRESTRequestMatcher::class)
            || !$container->hasDefinition('lexik_jwt_authentication.extractor.authorization_header_extractor')
        ) {
            return;
        }

        if (!$container->hasDefinition('lexik_jwt_authentication.extractor.authorization_header_extractor')) {
            return;
        }
```

Reply:
I changed the approach a bit, and instead of manipulating the service directly, I'm setting a parameter it relies upon.

This should make further use of changed authorization header easier to maintain (in case of more services) and cleans up this code.